docs/cunumeric/source/comparison/_comparison_generator.py

```diff
@@ -131,9 +163,7 @@ def _section(header, mod_ext, other_lib, klass=None, exclude_mod=None):
         "   Number of NumPy functions: {}".format(len(base_funcs)),
         "   Number of functions covered by "
         f"{other_lib}: {len(lg_funcs & base_funcs)}",
-        "   {} specific functions:".format(other_lib),
```

Comment:
why is this removed? is it relevant to the extension done in this pr?

Reply:
This was removed because this information is not used. Do you think I should put it back?

Reply:
No I was merely asking the logic behind removing it. If that line doesn't serve any purpose any more, I'm fine with removing it.